src/Functions/normalizeString.cpp

```diff
@@ -171,10 +171,116 @@ using FunctionNormalizeUTF8NFKD = FunctionStringToString<NormalizeUTF8Impl<Norma
 
 REGISTER_FUNCTION(NormalizeUTF8)
 {
-    factory.registerFunction<FunctionNormalizeUTF8NFC>();
-    factory.registerFunction<FunctionNormalizeUTF8NFD>();
-    factory.registerFunction<FunctionNormalizeUTF8NFKC>();
-    factory.registerFunction<FunctionNormalizeUTF8NFKD>();
+    FunctionDocumentation::Description description_nfc = R"(
+Normalizes a UTF-8 string according to the NFC normalization form.
```

Comment:
Wikipedia link is missing.

(same comment applies below too for other description fields)